articles/quickstart/backend/java-spring-security5/index.yml

```diff
@@ -0,0 +1,43 @@
+title: Spring Security 5 Java API
+thirdParty: false
+author:
+  name: Jim Anderson
+  email: jim.anderson@auth0.com
+  community: false
+alias:
+  - spring security
+  - spring
+languages:
+  - Java
+framework:
+  - Spring
+image: /media/platforms/java.png
+topics:
+  - quickstart
+contentType: tutorial
+useCase: quickstart
+articles:
+  - 01-authorization
+  - 02-using
+  - 03-troubleshooting
+show_steps: true
+github:
+  org: auth0-samples
+  repo: auth0-spring-security5-api-sample
+requirements:
+  - Java 8 or above
+next_steps:
+  - path: 01-authorization
+    list:
+    - text: Configure other identity providers
+      icon: 345
+      href: "/identityproviders"
+    - text: Enable multifactor authentication
+      icon: 345
+      href: "/multifactor-authentication"
+    - text: Learn about anomaly detection
+      icon: 345
+      href: "/anomaly-detection"
```

Comment:
any reason to list this one as related?

Reply:
No specific reason, other than that other API Quickstarts (and maybe all Quickstarts in general) appear to include it. Should we remove it from this one and/or include something else?

Reply:
I don't see a strong connection to the quickstart, but if others have it you can keep it. Maybe @albertoperdomo has a different view on this?